ut_core.py

```diff
@@ -0,0 +1,203 @@
+
+import yaml
```

Comment:
? I'm confused where this is coming from? you need to re-base your branch, this is incorrect and should be removed, your not adding this file.